.github/workflows/ci.yml

```diff
@@ -172,16 +213,45 @@ jobs:
         run: "echo RELEASE_VERSION=${GITHUB_REF:10} >> $GITHUB_ENV"
       - name: "Run Poetry Version"
         run: "poetry version $RELEASE_VERSION"
-      - name: "Install Dependencies (needed for mkdocs)"
```

Comment:
These few lines need to stay, this is responsible for packaging the docs to be available in the Nautobot UI. Looks like a cookiecutter PR is necessary.

Reply:
Fixed